src/cmd/fetch.zig

```diff
@@ -370,7 +370,15 @@ fn print_pkgs(alloc: std.mem.Allocator, w: std.fs.File.Writer, m: zigmod.Module)
         const ident = try zig_name_from_pkg_name(alloc, d.name);
         try w.print("    pub const {s} = package_data._{s};\n", .{ ident, d.id[0..12] });
     }
-    try w.writeAll("}");
+
+    try w.writeAll("    pub fn addAllTo(artifact: *std.build.LibExeObjStep) void {\n");
```

Comment:
please add a spacer `\n` so the function definition isnt scrunched up against the decl list

Reply:
feel free to add a comment too that the method is for gyro compat and doesnt call the full `deps.addAllTo(artifact);`